diff --git a/src/components/VideoPlayer.vue b/src/components/VideoPlayer.vue
index 5192669b..71e95011 100644
--- a/src/components/VideoPlayer.vue
+++ b/src/components/VideoPlayer.vue
@@ -901,4 +901,22 @@ html .shaka-range-element:focus {
     align-items: center;
     justify-content: center;
 }
+
+/* Modify play button to something more minimal */
+.shaka-play-button {
+    box-sizing: border-box;
+    padding: calc(15% / 2);
+    width: 0;
+    height: 0;
+    margin: 0;
+    border-radius: 50%;
+    box-shadow: rgba(0 0 0 / 10%) 0 0 0px 0;
+    border: none;
+    background-size: 70%;
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-color: rgba(255 255 255 / 0%);
+    opacity: 0;
+    transition-property: none;
+}
 </style>
