diff --git a/docker/nginx.conf b/docker/nginx.conf
index 3b7ef86b..b8b15183 100644
--- a/docker/nginx.conf
+++ b/docker/nginx.conf
@@ -1,11 +1,10 @@
 server {
-    listen 80;
-    listen [::]:80;
+    listen 8080;
+    listen [::]:8080;
     server_name localhost;
-    error_log off;
 
     location / {
-        root /usr/share/nginx/html;
+        root /opt/app-root/src;
         index index.html index.htm;
     }
 
